lib/controllers/status-bar-tile-controller.js

```diff
@@ -20,8 +21,13 @@ export default class StatusBarTileController extends React.Component {
     confirm: PropTypes.func.isRequired,
     repository: PropTypes.object.isRequired,
     toggleGitTab: PropTypes.func,
+    toggleGithubTab: PropTypes.func,
     ensureGitTabVisible: PropTypes.func,
   }
+  // todo: ensureGitTabVisible is unused.
```

Comment:
@kuychaco : do you have thoughts on this?

Reply:
Hmmm looks like we're never actually calling this method. So maybe we can just drop it?

Reply:
Oh oops, didn't see the comments below as well. Let's ask @smashwilson about the desired merge conflict behavior. Maybe this is a regression?